import-export-cli/cmd/aws/init.go

```diff
@@ -227,41 +230,42 @@ func getOAS() error {
 					fmt.Println(logsScannerGetExportCmd.Text())
 				}
 				if err := logsScannerGetExportCmd.Err(); err != nil {
-					fmt.Println("Error reading debug logs from standard error. (get-export command)", err)
+					utils.HandleErrorAndContinue("Error reading debug logs from standard error. (get-export command)", err)
 				}
 			}
-			
+
 			if utils.VerboseModeEnabled() {
 				outputScannerGetExportCmd := bufio.NewScanner(stdout)
 				for outputScannerGetExportCmd.Scan() {
 					fmt.Println(outputScannerGetExportCmd.Text())
 				}
 				if err := outputScannerGetExportCmd.Err(); err != nil {
-					fmt.Println("Error reading output from standard output. (get-export command)", err)
+					utils.HandleErrorAndContinue("Error reading output from standard output. (get-export command)", err)
 				}
 			}
 
 			err = getExportCmd.Wait()
 			if err != nil {
-				fmt.Println("Could not complete get-export command successfully.", err)
-			} 
+				os.RemoveAll(tmpDir)
+				utils.HandleErrorAndExit("Could not complete get-export command successfully.", err)
+			}
 			break
 		}
 	}
 	if !found {
-		fmt.Println("Unable to find an API with the name", apiName)
 		os.RemoveAll(tmpDir)
+		utils.HandleErrorAndExit("Unable to find an API with the name "+apiName, err)
 		os.Exit(1)
```

Comment:
I think No need of using `os.Exit` once you use `HandleErrorAndExit` here.

Reply:
ah i missed that.